src/ubuntu/18.04/crossdeps/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-coredeps
+
+# Install the base toolchain we need to build anything (clang, cmake, make and the like).
+RUN apt-get update \
+    && apt-get install -y \
+        apt-transport-https \
+        ca-certificates \
+        gnupg \
+        software-properties-common \
+        wget \
+    && wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc | apt-key add - \
+    && apt-add-repository 'deb https://apt.kitware.com/ubuntu/ xenial main' \
+    && apt-get update \
+    && apt-get install -y \
+        binfmt-support \
+        binutils-arm-linux-gnueabihf \
+        build-essential \
+        cmake \
+        debootstrap \
+        gdb \
+        make \
+        qemu \
+        qemu-user-static \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update \
+    && apt-add-repository 'deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-9 main' \
```

Comment:
This is not correct. This would try to install packages for xenial (ubuntu 16.04) into 18.04. You need to replace xenial by bionic here.
However, this is not needed anymore, as the llvm-9 and related tools are now available for Ubuntu 18.04 natively. So you can get rid of the lines 27..29 completely.